src/NServiceBus.Core/Pipeline/BehaviorContextStacker.cs

```diff
@@ -8,7 +8,7 @@
     /// <summary>
     /// A stack of <see cref="BehaviorContext"/>s.
     /// </summary>
-    public class BehaviorContextStacker : IDisposable
+    internal class BehaviorContextStacker : IDisposable
```

Comment:
I realise it's not the standard in the Particular org, but I'm :+1: for explicit access modifiers always.